.github/bin/build-and-test.sh

```diff
@@ -94,6 +94,7 @@ case "$MODE" in
     bazel coverage \
           --combined_report=lcov \
           --coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main \
+          --java_runtime_version=remotejdk_11 \
```

Comment:
Could this be the issue we see with the remotejdk complaints ?

Reply:
Yes. This is the newer syntax for 5.x and it does not work on 4.x. I've just reworked the CI so that the `default-jdk` package is used (i.e. a system-wide Java installation), see https://github.com/chipsalliance/verible/actions/runs/3696121095/jobs/6259392848.